mvpa2/cmdline/cmd_ttest.py

```diff
@@ -58,14 +61,35 @@ def setup_parser(parser):
                         belong to.  E.g. if values are accuracies, it would be the
                         'greater', if errors -- the 'less'""")
 
+def check_filetype(fn):
+    if '.nii' in fn:
```

Comment:
Well... Nibabel and thus we support not only nifti... Moreover I don't see a point to have if s here and then in the originating function really at this point... What about just checking for common extensions of hdf and then if not matched giving to nibabel?